sdk/src/packet.rs

```diff
@@ -24,6 +24,7 @@ pub struct Meta {
     pub v6: bool,
     pub seed: [u8; 32],
     pub slot: Slot,
+    pub tracer_tx_signature: Option<Signature>,
```

Comment:
Is this a non-trivial memory increase? Can we instead track a bool for if it's a tracer tx?

Reply:
yeah you're right. I was trying to avoid deserializing the signature again in BankingStage when reporting the metrics since Sigverify already owns the signature.

Reply:
The best thing I could think of is send a `Option<Box<Signature>>`  along with every Packet to BankingStage so that the Packet doesn't accrue extra memory, but the channel would have an extra `size_of` pointer for every packet.

Maybe I should just eat the cost of the extra deserialization in BankingStage...

Reply:
Another options is a separate channel just for tracer transactions that takes (Packet,Signature) tuples

Reply:
Extra deserialization in banking stage seems fine if you only do it for packets that have a `tracer` metadata flag